tests/test_utils/test_properties.py

```diff
@@ -73,7 +84,19 @@ def test_is_cached_property():
 
 def test_classproperty():
     """
-    Test the `is_cached_property` util.
+    Test the `classproperty` decorator.
+    """
+    assert ATest.classproperty == ATest.CLS_VALUE
```

Comment:
A succinct way of validating that `classproperty` recomputes is `assert id(ATest.classproperty) != id(ATest.classproperty)`